File: clients/python/examples/workflow_auto_launch.py
from __future__ import annotations

from sikuligo import Pattern, Screen

# Primary constructor: connect first, spawn fallback.
screen = Screen()
try:
    match = screen.click(Pattern("assets/pattern.png").exact())
    print(f"clicked match target at ({match.target_x}, {match.target_y})")
finally:
    screen.close()

File: packages/api/cmd/sikuli-go/examples/node/workflow-auto-launch.mjs
import { ensureSikuliGoOnPath } from "./bootstrap.mjs";
import { Screen, Pattern } from "@sikuligo/sikuli-go";

ensureSikuliGoOnPath();

// Primary constructor: connect first, then spawn fallback.
const screen = await Screen();
try {
  const match = await screen.click(Pattern("assets/pattern.png").exact());
  console.log(`clicked match target at (${match.targetX}, ${match.targetY})`);
} finally {
  await screen.close();
}
